### DISCUSSION TITLE
Basic plan has all features with limited storage space & More plan options

### DISCUSSION THREAD

**Discussion by trymeouteh** [streetwriters/notesnook#1138] [Ideas]
I would like to make a suggestion to the plans and what each plan offers. Essentially users get all of the features for free but with limited storage. Plans will just be for more storage space on the cloud.

1. Please consider having the free basic plan have all the features that are available in the pro plan but with limited storage (Such as 50MB of storage). This way those with free accounts get all the features, especially when sharing notes with others but to get a good amount of storage, they can pay for a plan.
2. More plan options. Have cheaper plans with less storage such as a 1GB storage plan, 5GB storage plan, etc. And the current Pro plan could be changed to have a limit on storage space such as 5GB.

I believe this is a more FOSS way of offering a service since all features are free, but if you want more storage space on a server, you got to pay for renting that storage space.

**Comment by thecodrr** [comment]
Please check other discussions before posting new ones. This is a duplicate of #719
